safenode/src/network/mod.rs

```diff
@@ -295,25 +295,57 @@ impl Network {
     /// Returns the closest peers to the given `XorName`, sorted by their distance to the xor_name.
     /// Excludes the client's `PeerId` while calculating the closest peers.
     pub async fn client_get_closest_peers(&self, xor_name: XorName) -> Result<Vec<PeerId>> {
```

Comment:
Does it make sense to add in `query_for` here too?

Reply:
Perhaps so, I had it first actually. But then I realised that clients should never call only local stored closest (or maybe only in rare optimisation cases?) because all (or very close to all) their interaction with network is to random places across it.

So I though that it should just be default, and no need to distinguish?